packages/types/dmProfile.cfc

```diff
@@ -87,15 +87,19 @@ TYPE PROPERTIES
 		ftType="list" ftDefaultType="evaluate" ftDefault="application.fapi.getConfig('general','locale')" 
 		ftListDataTypename="dmProfile" ftListData="getLocales"
 		hint="Profile object locale">
+		
+	<cfproperty name="timeFormat" type="string" required="yes" default="12h" 
+		ftSeq="14" ftFieldset="User settings" ftLabel="Time format" 
+		ftType="list" ftList="12h,24h" ftDefaultType="evaluate" ftDefault="application.fapi.getConfig('general','xxx', '12h')" />
```

Comment:
I'm guessing this default value is just a placeholder (there is no "xxx" property in the "general" config).

Should the default time format come from a config, or could it be determined from the users locale when the profile is created? If it's not worth the time, maybe it could simply default to "12h"?